space.fips.Fips.appdata.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2018 Matwey V. Kornilov <matwey.kornilov@gmail.com> -->
+<component type="desktop-application">
```

Comment:
The appdata file lacks some important information. See https://github.com/flathub/flathub/wiki/App-Requirements#appstream